include/view.h

```diff
@@ -389,8 +390,10 @@ void view_toggle_decorations(struct view *view);
 
 bool view_is_always_on_top(struct view *view);
 bool view_is_always_on_bottom(struct view *view);
+bool view_is_omnipresent(struct view *view);
```

Comment:
It doesn't seem like this is used anywhere so I'd suggest just removing `view_is_omnipresent`.